modules/bullet/collision_object_bullet.h

```diff
@@ -127,15 +128,18 @@ class CollisionObjectBullet : public RIDBullet {
 	VSet<RID> exceptions;
 
 	bool need_body_reload = true;
+	bool need_collision_fm_reload = true;
 
```

Comment:
More nitpicking, but I think "need" needs an "s": `needs_body_reload` and `needs_collision_filters_reload`.

Reply:
Changed